v2.0/information-schema.md

```diff
@@ -84,6 +84,22 @@ Column | Description
 `TABLE_NAME` | Name of the table containing the constraint.
 `REFERENCED_TABLE_NAME` | Name of the table containing the unique or primary key constraint that the foreign key constraint references.
 
+### role_table_grants
+
+The `role_table_grants` view identifies which [privileges](privileges.html) have been granted on tables or views where the grantor
+or grantee is a currently enabled role. This table is identical to the [table privileges](#table_privileges) view.
```

Comment:
nit: use underscore and backticks for `#table_privileges` link text:

```
[`table_privileges`](#table_privileges)
```

Reply:
@spencerkimball, I'll save you the time and make this change myself, as long as I'm allowed to push  to your branch.